lib/erlef/stipend_mail.ex

```diff
@@ -13,9 +13,15 @@ defmodule Erlef.StipendMail do
 
     subject_suffix = "on #{dt_str} via #{full_name(proposal)} (#{proposal.email_address}"
 
+    to =
+      case proposal.stipend_type do
+        "Elixir Outreach" -> {"Elixir Outreach Stipend Committee", "elixir_outreach@theeef.org"}
```

Comment:
```suggestion
        "Elixir Outreach" -> {"Elixir Outreach Stipend Committee", "elixir_outreach@erlef.org"}
```

Reply:
Are those the same? Because the one I tested was at theeef.org :)

Reply:
@josevalim if you just tested locally, it would have worked with an invalid domain 😄